src/features/schedule/ScheduleCardCourses.tsx

```diff
@@ -47,33 +58,46 @@ const meetingTimeCampusMap = (course: ICourseSchedule, meetingTime: IMeetingTime
-  let filteredCourses;
-  filteredCourses = course.attributes.meetingTimes.map(
-    (meetingTime: IMeetingTime) =>
-      !isMidterm(meetingTime) && (
-        <ListItem key={`${course.id}${meetingTime.beginDate}${meetingTime.beginTime}`}>
-          <ListItemContent>
-            {courseItemLeadText(course.attributes.courseSubject, course.attributes.courseNumber)}
-            <ListItemText>
-              <ListItemDescription fontSize={theme.fontSize[16]} color={Color['neutral-700']}>
-                {course.attributes.scheduleDescription} &bull; {meetingTime.room}{' '}
-                {meetingTime.buildingDescription}
-              </ListItemDescription>
-              <ListItemDescription>
-                {formatTime(meetingTime.beginTime)} - {formatTime(meetingTime.endTime)}
-              </ListItemDescription>
-            </ListItemText>
-            {courseOnCorvallisCampus(course) && meetingTimeCampusMap(course, meetingTime)}
-          </ListItemContent>
-        </ListItem>
-      )
-  );
-  return filteredCourses;
-};
-
 const ScheduleCardCourses = (props: ScheduleCardCoursesProps) => {
   const { selectedCourses } = props;
+  const [isOpen, setOpen] = useState(false);
+  const [courseAttributes, setCourseAttributes] = useState<ICourseScheduleAttributes | null>(null);
+  const meetingTimeListItems = (course: ICourseSchedule): JSX.Element[] => {
+    let filteredCourses;
+    filteredCourses = course.attributes.meetingTimes.map(
+      (meetingTime: IMeetingTime) =>
+        !isMidterm(meetingTime) && (
+          <ListItem key={`${course.id}${meetingTime.beginDate}${meetingTime.beginTime}`}>
+            <ListItemContentButton
+              onClick={() => {
+                toggleCourse(course.attributes);
+                Event('courses', 'course clicked', course.attributes.courseTitle);
+              }}
+            >
+              {courseItemLeadText(course.attributes.courseSubject, course.attributes.courseNumber)}
+              <ListItemText>
+                <ListItemDescription fontSize={theme.fontSize[16]} color={Color['neutral-700']}>
+                  {course.attributes.scheduleDescription} &bull; {meetingTime.room}{' '}
+                  {meetingTime.buildingDescription}
+                </ListItemDescription>
+                <ListItemDescription>
+                  {formatTime(meetingTime.beginTime)} - {formatTime(meetingTime.endTime)}
+                </ListItemDescription>
+              </ListItemText>
+              {courseOnCorvallisCampus(course) && meetingTimeCampusMap(course, meetingTime)}
+            </ListItemContentButton>
+          </ListItem>
+        )
+    );
+    return filteredCourses;
+  };
+
+  // Hides or shows course details
+  const toggleCourse = courseAttributes => {
+    setOpen(!isOpen);
+    setCourseAttributes(courseAttributes);
+  };
+
   return (
     <CardSection>
       {/* TODO: course should NOT be a link */}
```

Comment:
Remove old comment?

Reply:
not sure what this means? Are you saying ditch the comment the line above on 95?

Reply:
nvm, i see what we are looking at. I was focused on the green and missed line 79. Duh. I didn't write it, but i don't think it applies anymore so i'll remove it.